crates/bevy_render/src/render_resource/texture.rs

```diff
@@ -7,8 +7,22 @@ pub struct TextureId(Uuid);
 
 /// A GPU-accessible texture.
 ///
-/// May be converted from and dereferences to a wgpu [`Texture`](wgpu::Texture).
-/// Can be created via [`RenderDevice::create_texture`](crate::renderer::RenderDevice::create_texture).
+/// Although primarily used to to store data that "texture" the surfaces of solids, textures can also be used as general purpose
```

Comment:
Maybe something like `Although primarily used to store image data that is applied (mapped) to the surface of polygon`
Using "texture" to help describe what texture is feels a bit funny. Also, there aren't really solids in this context. Faces, fragments, polygons, triangles, etc...

Reply:
@DGriffin91 thanks! Rob submitted some suggested changes which I think address this. How do you feel about how it is now?

Reply:
Yeah, this works fine.